autogen/versions.tf

```diff
@@ -17,3 +17,13 @@
 terraform {
   required_version = ">= 0.12"
 }
+
+{% if beta_cluster %}
+provider "google-beta" {
```

Comment:
We should not specify a version in the module itself.

Instead, we should simply specify [required versions](https://github.com/forseti-security/terraform-google-forseti/blob/master/versions.tf#L20).